example/client_tickets/bootstrap.php

```diff
@@ -46,7 +47,7 @@ function client_ticket_list(Parameter\Source\Client $client, Parameter\Plugin $p
 	));
 
 	if (empty($tickets)) {
-		Error\Exception(I18nf('No %s found', I18n('tickets')));
+		throw new SDKException(I18nf('No %s found', I18n('tickets')));
```

Comment:
Since we already use the namespace `UbersmithSDK\Error`. I think it'd be cleaner to change to line to
```php
throw new Error\SDKException(I18nf('No %s found', I18n('tickets')));
```
And remove `line 10` you added. This can be seen in other ubersmith plugin we've built in the past as well.